.gitignore

```diff
@@ -1,4 +1 @@
-.idea
-dist
-envelope.egg-info
-__pycache__
+tests/gpg_ring/random_seed
```

Comment:
Putting random_seed here is great!

I'd prefer them to stay here:
 	
dist
envelope.egg-info
__pycache__

(these dirs appears when running tests or deploying for every repository user)